donkeycar/parts/tfmini.py

```diff
@@ -0,0 +1,72 @@
+import time
+import serial
+import logging
+
+class TFMini:
+    """
+    Class for TFMini and TFMini-Plus distance sensors.
+    See wiring and installation instructions at https://github.com/TFmini/TFmini-RaspberryPi
+
+    Returns distance in centimeters. 
+    """
+
+    def __init__(self, port="/dev/serial0", baudrate=115200, poll_delay=0.01, init_delay=0.1):
+        self.ser = serial.Serial(port, baudrate)
+        self.poll_delay = poll_delay
+
+        self.dist = 0
+
+        if not self.ser.is_open:
+            self.ser.close() # in case it is still open, we do not want to open it twice
+            self.ser.open()
+
+        self.logger = logging.Logger("TFMini Logger")
```

Comment:
Can you please move the logger outside of the class into the module scope as we have done for most of the other modules. You create the logger with:
```python
logger = logging.getLogger(__name__)
```